egs/thchs30/s5/run.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+. ./cmd.sh ## You'll want to change cmd.sh to something that will work on your system.
+           ## This relates to the queue.
+. ./path.sh
+
+H=`pwd`  #exp home
+n=8      #parallel jobs
+
+thchs=/nfs/public/materials/data/thchs30-openslr
+#corpus and trans directory
+#you can obtain the database by uncommting the following lines
+#( cd $thchs
+#     echo "downloading THCHS30 at $PWD ..."
+#     wget http://www.openslr.org/resources/18/data_thchs30.tgz
+#     wget http://www.openslr.org/resources/18/resource.tgz
+#     tar xvf data_thchs30.tgz && tar xvf resource.tgz 
+#)
+
```

Comment:
Agree. the Librispeech way is really good. I duplicated the script and made some modifications to support thchs30. Another change is that we now only use the standard noisy test data, so all the data packages are downloaded together.